src/main/java/uk/gov/pay/directdebit/partnerapp/resources/GoCardlessAppConnectAccountTokenResource.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.pay.directdebit.partnerapp.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+import uk.gov.pay.directdebit.partnerapp.api.GoCardlessAppConnectAccountTokenRequestValidator;
+import uk.gov.pay.directdebit.partnerapp.services.GoCardlessAppConnectAccountService;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("/")
+public class GoCardlessAppConnectAccountTokenResource {
+    private static final Logger LOGGER = PayLoggerFactory.getLogger(GoCardlessAppConnectAccountTokenResource.class);
+
+    public static final String CODE_FIELD = "code";
+    public static final String STATE_TOKEN_FIELD = "state_token";
```

Comment:
I think these can just be inlined

Reply:
I would inline them as well, but they are used in test and Validator as well. I am keeping them in here because if they change then they can be updated easier if in one place only.